BookA-Pset_GeoObsBoreholeGeophysicalInterval

larswik opened the pull request:
# Pset: Pset_GeoObsBoreholeGeophysicalInterval

## Applicability: IfcGeoScienceObservation/BOREHOLELOG

## Description: 

Properties for describing geopysical intervals

| Property | Datatype | Description |
|----------|----------|-------------|
|PwaveVelocity| According to existing definition | According to existing definition |
|SwaveVelocity| According to existing definition | According to existing definition |
|YoungModulus|According to existing definition| According to existing definition|
|PoissonRatio|According to existing definition| According to existing definition |
|DryBulkDensity|IfcMassDensityMeasure| Ratio of the mass of solid to the unit total volume (dry material).|
|WetBulkDensity|IfcMassDensityMeasure|Ratio of the total mass to the unit total volume (material at its natural moisture content). |
|Resistivity| According to existing definition | According to existing definition |
|SpontaneousPotential|IfcElectricVoltageMeasure|Naturally occurring (static) electrical potential in the Earth. Spontaneous potentials are usually caused by charge separation in clay or other minerals, by the presence of a semipermeable interface impeding the diffusion of ions through the pore space of rocks, or by natural flow of a conducting fluid (salty water) through the rocks. Variations in SP can be measured in the field and in wellbores to determine variations of ionic concentration in pore fluids of rocks.|
|GroundwaterTemperature|IfcThermodynamicTemperatureMeasure|Groundwater temperature observed at specific location or expected in the subject section.|
|HoleDiameter|IfcPositiveLengthMeasure|Drilled Borehole diameter (mm)|
|GammaRay|IfcReal|Naturally occurring gamma radiation which characterizes the rock or sediment in a borehole or drill hole, measured by a gamma ray logging.|

larswik committed:
BookA-Pset_GeoObsBoreholeGeophysicalInterval

larswik committed:
Update Documentation.md

larswik committed:
Update Documentation.md

larswik committed:
Update Documentation.md

larswik committed:
Update Documentation.md

SergejMuhic commented on IFC4x3/Properties/b/BulkDensity_0nEw_cmQP96hsnDkHk7anm/Documentation.md:
```diff
@@ -0,0 +1 @@
+The bulk density of a rock is ?B = WG/VB, where WG is the weight of grains (sedimentary rocks) or crystals (igneous and metamorphic rocks) and natural cements, if any, and VB is the total volume of the grains or crystals plus the void (pore) space. The density can be dry if the pore space is empty, or it can be saturated if the pores are filled with fluid (e.g., water), which is more typical of t ...
```
What does "?B" mean?
This is also odd "(e.g., water)".
"The density can be dry if the pore space is empty, or it can be saturated if the pores are filled with fluid" -> this does not make sense since the WG does not consider the weight (mass?) of water.

larswik replied:
Changed in [653d69b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/583/commits/653d69bb071ec1b43b5e0983bba3cee875cd0342) to DryBulkDensity with description "Ratio of the mass of solid to the unit total volume (dry material)." and WetBulkDensity with description "Ratio of the total mass to the unit total volume (material at its natural moisture content)." (from Pset_GeoScienceProperties - see #632 which needs to be synced with this)

SergejMuhic commented on IFC4x3/Properties/b/BulkDensity_0nEw_cmQP96hsnDkHk7anm/Documentation.md:
```diff
@@ -0,0 +1 @@
+The bulk density of a rock is ?B = WG/VB, where WG is the weight of grains (sedimentary rocks) or crystals (igneous and metamorphic rocks) and natural cements, if any, and VB is the total volume of the grains or crystals plus the void (pore) space. The density can be dry if the pore space is empty, or it can be saturated if the pores are filled with fluid (e.g., water), which is more typical of t ...
```
```suggestion
The bulk density of a rock is ?B = WG/VB, where WG is the weight of grains (sedimentary rocks) or crystals (igneous and metamorphic rocks) and natural cements, if any, and VB is the total volume of the grains or crystals plus the void (pore) space. The density can be dry if the pore space is empty, or it can be saturated if the pores are filled with fluid (e.g., water) which is more typical of the subsurface (in situ) situation. 
```

larswik replied:
See previous comment.

SergejMuhic commented on IFC4x3/Properties/e/Emodulus_0Tfz4hnAXCMOBE9sK0qAiG/DocProperty.xml:
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
`YoungModulus`? I would re-use.

larswik replied:
Re-used in [653d69b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/583/commits/653d69bb071ec1b43b5e0983bba3cee875cd0342)

SergejMuhic commented on IFC4x3/Properties/g/GammaRay_0uQOo2p7L7gfdVTXwVzLwb/DocProperty.xml:
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GammaRay_0uQOo2p7L7gfdVTXwVzLwb" Name="GammaRay" UniqueId="38698c82-cc75-47aa-99df-761e9ff55ea5" PrimaryDataType="IfcReal" />
```
`IfcReal`?

larswik replied:
@j-weil: Which measure/unit? API?

larswik replied:
Not changed.

SergejMuhic commented on IFC4x3/Properties/g/GroundWaterTemperature_1$VLyCC_rAj9xz37v0OTla/Documentation.md:
```diff
@@ -0,0 +1 @@
+Groundwater temperature obtained by a tempereature logging.
```
I would like to correct this but I do not understand what it means.

larswik replied:
Re-used definition in [653d69b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/583/commits/653d69bb071ec1b43b5e0983bba3cee875cd0342)

SergejMuhic commented on IFC4x3/Properties/h/HoleDiameter_2CRBoJG6P9xwWyupBQZ76M/Documentation.md:
```diff
@@ -0,0 +1 @@
+Drilled Borehole diameter.
```
```suggestion
Drilled borehole diameter.
```

SergejMuhic commented on IFC4x3/Properties/h/HoleDiameter_2CRBoJG6P9xwWyupBQZ76M/DocProperty.xml:
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Probably doesn't make sense to have it here also.

larswik replied:
Kept and added sentence to definition according to domain proposal.

SergejMuhic commented on IFC4x3/Properties/g/GroundWaterTemperature_1$VLyCC_rAj9xz37v0OTla/DocProperty.xml:
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
I think I have seen this somewhere.

larswik replied:
Re-used from Pset_GeoAspects in [653d69b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/583/commits/653d69bb071ec1b43b5e0983bba3cee875cd0342)
- Pset_GeoAspects (merged)
- Pset_GeoObsPtObservationCommon (merged - used the same as Pset_GeoAspects in the referenced commit)
-Pset_GeoSciencePropertiesCommon (not merged - #632). After this PR:s have been merged, this one can be synced and the extra GroundwaterTemperature be deleted.

larswik replied:
Kept the property here.

SergejMuhic commented on IFC4x3/Properties/p/PoissonRatio_2y5cHUFyL0rhO1Q97aJWbg/DocProperty.xml:
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PoissonRatio_2y5cHUFyL0rhO1Q97aJWbg" Name="PoissonRatio" UniqueId="bc16645e-3fc5-40d6-b601-6891e44e096a" PrimaryDataType="IfcRatioMeasure" />
```
Propose re-using `PoissonRatio`.

larswik replied:
Re-used in [653d69b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/583/commits/653d69bb071ec1b43b5e0983bba3cee875cd0342)

SergejMuhic commented on IFC4x3/Properties/s/SpontaneousPotential_0QWHw4BW90SxlAkUg4JefW/Documentation.md:
```diff
@@ -0,0 +1 @@
+Naturally occurring (static) electrical potential in the Earth. Spontaneous potentials are usually caused by charge separation in clay or other minerals, by the presence of a semipermeable interface impeding the diffusion of ions through the pore space of rocks, or by natural flow of a conducting fluid (salty water) through the rocks. Variations in SP can be measured in the field and in wellbores ...
```
```suggestion
Naturally occurring (static) electrical potential in the Earth. Spontaneous potentials (SP) are usually caused by charge separation in clay or other minerals, by the presence of a semipermeable interface impeding the diffusion of ions through the pore space of rocks, or by natural flow of a conducting fluid (salty water) through the rocks. Variations in SP can be measured in the field and in wellbores to determine variations of ionic concentration in pore fluids of rocks.
```

SergejMuhic commented on IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_GeoObsBoreholeGeophysicalInterval/Documentation.md:
```diff
@@ -0,0 +1 @@
+Properties for describing geopysical intervals.
```
```suggestion
Properties for describing geophysical intervals.
```

larswik committed:
Update IFC4x3/Properties/s/SpontaneousPotential_0QWHw4BW90SxlAkUg4JefW/Documentation.md

Co-authored-by: SergejMuhic <sergejs1@gmail.com>

larswik committed:
Update IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_GeoObsBoreholeGeophysicalInterval/Documentation.md

Co-authored-by: SergejMuhic <sergejs1@gmail.com>

larswik committed:
Merge remote-tracking branch 'upstream/tunnel' into feature/BookA-Pset_GeoObsBoreholeGeoPhysicalInterval

larswik committed:
Fixed according to (some comments)

Including re-using GroundwaterTemperature also in Pset_GeoObsPtObservationCommon.

larswik committed:
Merge remote-tracking branch 'upstream/tunnel' into feature/BookA-Pset_GeoObsBoreholeGeoPhysicalInterval

larswik committed:
Merge remote-tracking branch 'upstream/tunnel' into feature/BookA-Pset_GeoObsBoreholeGeoPhysicalInterval

larswik committed:
Fixed after discussions with domain

+ added lost properties

larswik committed:
Reused the exiswting PoissonRatio

SergejMuhic commented on IFC4x3/Properties/d/DryBulkDensity_0nEw_cmQP96hsnDkHk7anm/Documentation.md:
```diff
@@ -0,0 +1 @@
+Ratio of the mass of solid to the unit total volume (dry material).
```
```suggestion
Ratio of the mass of the solid to the unit total volume (dry material).
```

SergejMuhic committed:
Update IFC4x3/Properties/d/DryBulkDensity_0nEw_cmQP96hsnDkHk7anm/Documentation.md
